controls/check_test.rb

```diff
@@ -0,0 +1,35 @@
+title 'Tests to confirm check works as expected'
+
+plan_name = input('plan_name', value: 'check')
+plan_ident = "#{ENV['HAB_ORIGIN']}/#{plan_name}"
+hab_path = input('hab_path', value: 'hab')
+
+control 'core-plans-check' do
+  impact 1.0
+  title 'Ensure check works'
+  desc '
+  We confirm that the check executable being present in the expected location.
+
+  We also run the following to ensure checkmk is correctly packaged:
+    $ ls -al <pkg>/lib/libcheck.so
+    $ ls -al <pkg>/bin/checkmk
+  '
+  expect_pkg_ident = command("#{hab_path} pkg path #{plan_ident}")
```

Comment:
Did you mean for these to be 'expect_pkg_ident' or 'check_pkg_ident' or just 'pkg_ident'?